app/components/status-badge.jsx

```diff
@@ -0,0 +1,66 @@
+// https://github.com/EnCiv/civil-pursuit/issues/47
+
+'use strict'
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import cx from 'classnames';
+
+function StatusBadge(props) {
+    const { style = {}, className = "", name = "", status = "", number = undefined, ...otherProps } = props;
+    const classes = useStyles();
+
+    const statusMapping = {
+        progress: "Unread",
+        complete: "Complete",
+        inactive: "Group, Inactive",
+        error: "Most Important"
+    }
+
+    return (
+        <span className={cx(classes.container, classes[status.toLowerCase()])}>
```

Comment:
- [ ] need to add className as another param to cx() so that the className passed down from the parent is added to the className for this span.

- [ ] need to add {...otherProps} to the span so that any props like style or onClick passed from the parent are added to this span.